src/test/scala-2.11/markets/participants/strategies/TestRandomTradingStrategy.scala

```diff
@@ -0,0 +1,85 @@
+package markets.participants.strategies
+
+import akka.agent.Agent
+
+import markets.tickers.Tick
+import markets.tradables.Tradable
+
+import scala.collection.mutable
+import scala.util.Random
+
+
+class TestRandomTradingStrategy(val prng: Random)
+  extends RandomTradingStrategy {
+
+  import TestRandomTradingStrategy._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2424830954/issues/source?bid=2921361&fileBranchId=3239634#l15)